docs/validator.md

```diff
@@ -0,0 +1,33 @@
+@typedef {function} can-validate/types/validator validator
+@parent can-validate/types
+
+A function that validates a value against predefined constraints and returns validation errors, if any are found.
+
+@signature `validator(value)`
+
+@param {Boolean|String|Number} value A simple value to validate
```

Comment:
`@param {*}`  should be used because anything can be passed.  An Object could be passed too.